build/files

```diff
@@ -297,7 +297,16 @@ QT_HDR =
     wx/qt/toolbar.h
     wx/qt/tooltip.h
     wx/qt/toplevel.h
-    wx/qt/window.h
+		wx/qt/treectrl.h
```

Comment:
Wrong indentation, please check for hard TABs.

Reply:
Fixed.